README.md

```diff
@@ -57,12 +57,22 @@ For example, to set the `max-width` to `80rem`:
 </script>
 ```
 
-If you're importing;
+- `hiring`: A link to a hiring page which will display a link in the header to the job page (default: `false`)
 
-```js
-import { createNav } from '@canonical/global-nav';
+For example, to set the `hiring` to `https://boards.greenhouse.io/canonical/jobs/1586585`
+
+````html
+<script src="/node_modules/@canonical/global-nav/dist/index.js"></script>
+
+<script>
+  canonicalGlobalNav.createNav({
+    hiring: 'https://boards.greenhouse.io/canonical/jobs/1586585',
+  });
+</script>
+
+If you're importing; ```js import { createNav } from '@canonical/global-nav';
```

Comment:
I think there should be a code end block "```"  above this line and the new block that starts on this line should be moved to the next line.

Reply:
done